### DISCUSSION TITLE
PROBLEM COMING WITH THE DISTRO

### DISCUSSION THREAD

**Discussion by shailpujan88** [camicroscope/caMicroscope#769] [General]
I have followed the instructions and cloned the project and when i run the command in distro it is showing this :
<img width="768" alt="image" src="https://github.com/camicroscope/caMicroscope/assets/114280481/94c9b58b-88e9-4a1c-a829-615b5582adfa">

**Comment by birm** [comment]
I don't see an error, just it being a bit verbose with logs. Is everything up as expected ?( try running `docker ps`)
